packages/core/contracts/common/implementation/Multicall.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.5.0;
```

Comment:
we already have this implementation here: https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/common/implementation/Multicall3.sol

which should be used over this implementation I think. the deployment address can be found here: https://github.com/mds1/multicall#multicall3-contract-addresses

Reply:
Oh, sorry I thought they were different implementations. Fixed: switched to Muticall3